proc.c

```diff
@@ -1877,8 +1877,9 @@ rb_mod_public_instance_method(VALUE mod, VALUE vid)
  *
  *  Defines an instance method in the receiver. The _method_
  *  parameter can be a +Proc+, a +Method+ or an +UnboundMethod+ object.
- *  If a block is specified, it is used as the method body. This block
- *  is evaluated using #instance_eval.
+ *  If a block is specified, it is used as the method body.
+ *  If a block or a method has parameters, they're used as method parameters.
```

Comment:
What does "or a method" mean?

Reply:
What I wanted to mention is `method` argument. I think it should be clear. I'll fix it.

Reply:
pipes?